samples/_opengl/ObjLoader/src/ObjLoaderApp.cpp

```diff
@@ -93,7 +93,7 @@ void ObjLoaderApp::writeObj()
 	fs::path filePath = getSaveFilePath();
 	if( ! filePath.empty() ) {
 		console() << "writing mesh to file path: " << filePath << std::endl;
-		writeObj( writeFile( filePath ), mMesh );
+		ci::writeObj( writeFile( filePath ), mMesh );
```

Comment:
I know, but this one is necessary or the sample doesn't compile, since writeObj() is already defined as a function of the app.